### PR TITLE
Migrate html component

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Migrate html component

Use angular library for codemirror
Use debounce directive for updates

[stage-17]

## Motivation and Context
Angular migration epic

## How Has This Been Tested?
Tested locally.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: src/app/template-editor/template-components/html/html.component.ts [hunk 1]
```diff
@@ -1,17 +1,52 @@
...
+      }
+    });
   }
 
+  _load() {
+    var html = this.attributeDataFactory.getAvailableAttributeData(this.componentId, 'html');
+
+    this.html = html;
+
+    setTimeout(() => {
```
Comment (ezequielc) [inline src/app/template-editor/template-components/html/html.component.ts:37]: @alex-deaconu I am curious, with this new library, was this setTimeout still necessary? Thanks!

**Review by ezequielc — APPROVED** [review]

File: src/app/template-editor/template-components/html/html.component.ts (hunk 1 above)
Comment (alex-deaconu) [reply src/app/template-editor/template-components/html/html.component.ts:37]: @ezequielc unfortunately yes, I tested by just initializing the object with those properties and it didn't seem to work correctly. I did not spend time to investigate further however. Thanks.

Comment (ezequielc) [reply src/app/template-editor/template-components/html/html.component.ts:37]: @alex-deaconu Sounds good, no problem. Thanks!

**Pull request merged**